web/libs/datamanager/src/components/MainView/GridView/GridView.jsx

```diff
@@ -23,14 +23,22 @@ const GridHeader = observer(({ row, selected }) => {
   );
 });
 
-const GridBody = observer(({ row, fields }) => {
+export const GridBody = observer(({ row, fields }) => {
   const dataFields = fields.filter((f) => f.parent?.alias === "data");
 
   return dataFields.map((field, index) => {
     const valuePath = field.id.split(":")[1] ?? field.id;
-    const value = getProperty(row, valuePath);
+    const field_type = field.currentType;
+    let value = getProperty(row, valuePath);
 
-    return <GridDataGroup key={`${row.id}-${index}`} type={field.currentType} value={value} field={field} row={row} />;
+    /**The value is an array...
+     * In this case, we take the first element of the array
+     */
+    if (Array.isArray(value)) {
```

Comment:
should we also check the length of the array prior to retrieving its first element, to avoid a potential crash?

Reply:
It's not necessary Jo, since we are using getProperty to get value, if the array is empty it's returning undefined and it's not getting into this if